content/syllabus-written-assignments/file-directory-naming/_index.md

```diff
@@ -15,11 +15,13 @@ This assessment is designed to not only assess your current knowledge but also p
 
 ## Questions
 
-1. What is the recommended naming convention for files and directories in the programming language you are learning? Why is it important to follow this convention?
+1. What naming convention is recommended for files and directories in the programming language you're learning?
 
-2. What are some of the common mistakes that beginners make when naming files and directories? How can you avoid these mistakes?
+2. Why is it of importance to apply naming conventions when arranging files and directories?
```

Comment:
How about rephrasing this to: `Why is it important to use recommended naming conventions?`